### ISSUE TITLE
Create a train schedule building menu that is activated from the train

### ISSUE THREAD

**Issue by LevFendi** [Factorio-Access/FactorioAccess#138]

**Comment by ahicks92** [comment]
2.0 adds train groups and train interrupts.  Between the other problems not having this UI makes both of those hard to impossible to support.

**Comment by LevFendi** [comment]
Yeah, we will need this for sure, but probably using the new UI framework rather than the existing one?
